### DISCUSSION TITLE
We're sorry, but something went wrong on our end. on Orner account

### DISCUSSION THREAD

**Discussion by nanakochi123456** [mastodon/mastodon#27071] [General]
![image](https://github.com/mastodon/mastodon/assets/703000/49eda56e-c3bd-47fa-9423-20c50c26848f)

Only the owner account will get an error.
Admin, Moderator and user accounts are fine.

I am updating from v4.1.7 to v4.2.0.

ruby : 3.2.2
Errorlog: none

**Comment by cham2019** [comment]
Same here, please help

**Comment by coreice** [comment]
Did your precompilation step (`RAILS_ENV=production bundle exec rails assets:precompile`) went successful?

**Reply by nanakochi123456** [reply]
Was successful

mastodon@neetdon:~/live$ RAILS_ENV=production bundle exec rails assets:precompile
Everything's up-to-date. Nothing to do
mastodon@neetdon:~/live$

**Reply by nanakochi123456** [reply]
First step it commandline is successful.
SKIP_POST_DEPLOYMENT_MIGRATIONS=true RAILS_ENV=production bundle exec rails db:migrate

But 2nd step or later is failure

stodon@neetdon:~/live$ SKIP_POST_DEPLOYMENT_MIGRATIONS=true RAILS_ENV=production bundle exec rails db:migrate
I, [2023-09-23T22:48:52.243361 #1063044]  INFO -- : Migrating to CreateGlobalFollowRecommendations (20230818141056)
== 20230818141056 CreateGlobalFollowRecommendations: migrating ================
[strong_migrations] DANGER: No lock timeout set
-- create_view(:global_follow_recommendations, {:materialized=>{:no_data=>true}})
rails aborted!
StandardError: An error has occurred, this and all later migrations canceled:

**Reply by nanakochi123456** [reply]
Sorry... failed.

mastodon@neetdon:~/live$  RAILS_ENV=production bundle exec rake tmp:cache:clear
mastodon@neetdon:~/live$  RAILS_ENV=production bundle exec rails assets:precompile
Compiling...
Compiled all packs in /home/mastodon/live/public/packs
`isModuleDeclaration` has been deprecated, please migrate to `isImportOrExportDeclaration`
    at isModuleDeclaration (/home/mastodon/live/node_modules/babel-plugin-lodash/node_modules/@babel/types/lib/validators/generated/index.js:2740:35)
    at PluginPass.Program (/home/mastodon/live/node_modules/babel-plugin-lodash/lib/index.js:102:44)

mastodon@neetdon:~/live$

**Reply by coreice** [reply]
You may want to check my thread: https://github.com/mastodon/mastodon/discussions/27087

**Reply by nanakochi123456** [reply]
I increased the memory of the virtual machine to 7G, but the result is as follows
There was nearly 2G of free memory during compile.

mastodon@neetdon:~/live$ RAILS_ENV=production bundle exec rake tmp:cache:clear
mastodon@neetdon:~/live$ RAILS_ENV=production bundle exec rails assets:precompile
Compiling...
Compiled all packs in /home/mastodon/live/public/packs
`isModuleDeclaration` has been deprecated, please migrate to `isImportOrExportDeclaration`
    at isModuleDeclaration (/home/mastodon/live/node_modules/babel-plugin-lodash/node_modules/@babel/types/lib/validators/generated/index.js:2740:35)
    at PluginPass.Program (/home/mastodon/live/node_modules/babel-plugin-lodash/lib/index.js:102:44)

**Comment by cham2019** [comment]
not trying to hijack OP thread, but my issue is similar.  Same "We're sorry" error page when I try to see my Admin/Owner account.  From outside, without login into mastodon, page loads perfectly fine.  All mastodon services are running fine and up.
Also, I tried:  " RAILS_ENV=production bundle exec rails assets:precompile " and seems to me that it run successful.  I tried again today, and "nothing to update" message appeared.  Not sure what happened.  It just started when I upgraded to 4.2.0  ( 4.1.9 was running just fine )
![image](https://github.com/mastodon/mastodon/assets/48077863/85008606-55ce-43b0-b76b-7f2b8c8ac4a5)
